egs2/jibo_kids/asr1/conf/tuning/train_asr_wavlm_transformer.yaml

```diff
@@ -0,0 +1,89 @@
+freeze_param: [
```

Comment:
Probably good to reduce to 1 config for the initial commit (unless you are having multiple checkpoints uploaded to huggingface for better references)